core-bundle/src/Resources/contao/library/Contao/Automator.php

```diff
@@ -238,6 +238,8 @@ public function purgeTempFolder()
 	 */
 	public function purgeRegistrations()
 	{
+		@trigger_error(sprintf('Using "%s" has been deprecated and will be removed in Contao 5.0. Use MemberModel::findExpiredRegistrations() to find and delete expired registrations instead.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
Please also add an `@deprecated` annotation in the phpDoc comment.

Reply:
Done